x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -135,7 +136,7 @@ export class TaskRunnerFactory {
           if (isRetryableBasedOnAttempts) {
             // In order for retry to work, we need to indicate to task manager this task
             // failed
-            throw new ExecutorError(e.message, {}, true);
+            throw throwRetryError(new ExecutorError(e.message, {}, true), true);
```

Comment:
since we're no longer checking for instance of `ExecutorError` in the `getRetry` function, wondering if we even need this custom error class? Seems like you just use a normal error since you're passing the retry value to throw a retryable error.

Reply:
I think that will work! Resolved in this commit: https://github.com/elastic/kibana/commit/3f5fbd14c06b90082cad8c953e018a16213cf615